packages/ozone/src/daemon/blob-diverter.ts

```diff
@@ -0,0 +1,229 @@
+import {
```

Comment:
Just to toss out an idea— the `blob_push_event` has an `eventType`.  Could we make blob diversions a new event type and avoid needing to add a new table, new part of the daemon that needs the polling machinery, etc.?  I think the `blob_push_event` is already setup decently well for this situation, may just need to handle the `eventType`.  I could see most of the `BlobDiverter` class sticking around to encapsulate the specifics about how the blob is pushed out, but not need to handle the polling, etc.

Reply:
yeah I like that approach better, refactored the code to use that table instead.